commands/Fun/say.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2017-2018 dirigeants. All rights reserved. MIT license.
+const { Command } = require('klasa');
+
+module.exports = class extends Command {
+
+	constructor(...args) {
+		super(...args, {
+			description: 'Make the bot say something',
+			name: 'say',
+			usage: '<message:string>',
+			disableEveryone: true,
```

Comment:
Message send option NOT COMMAND option /tableflip

Reply:
I don't know how to add a message send option :l

Reply:
`message.channel.send(content, { disableEveryone: true })`

Reply:
Resolved